pkg/client/client.go

```diff
@@ -0,0 +1,72 @@
+package client
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/client-go/rest"
+)
+
+// ObjectKey identifies a Kubernetes Object.
+type ObjectKey types.NamespacedName
+
+// Client uses methods defined in runtime.Object to determine the REST endpoint
+// for Kubernetes resources.
```

Comment:
Why is this comment dangling? Does it end-up in the godoc? What's "Client"?

Reply:
Aah... I forgot to take it out. This is out of place. I need to put it somewhere but in context.